feature-libs/pickup-in-store/components/container/set-preferred-store/set-preferred-store.module.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { I18nModule } from '@spartacus/core';
+import {
+  IconModule,
+  OutletPosition,
+  provideOutlet,
+} from '@spartacus/storefront';
+import { SetPreferredStoreComponent } from './set-preferred-store.component';
+
+@NgModule({
+  imports: [CommonModule, IconModule, I18nModule],
+  exports: [SetPreferredStoreComponent],
+  declarations: [SetPreferredStoreComponent],
+  providers: [
+    // TODO we may need to turn this into an enum. Don't want to to though as then the module which doesn't contain the enum, becomes dependent on the one that does
```

Comment:
Can create this enum in storefinder's root. Now, bopis depends on storefinder.

Reply:
Done